composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
+        "doctrine/annotations": "^1.2",
```

Comment:
> I can only see this causing future dependency conflicts.

 I think it's fine in the case PHP-CS-Fixer is used as a Composer dependency: PHP <5.6 users will have `doctrine/annotations` 1.2 installed while PHP 5.6+ users will have version 1.3+ which is supposed to be backward compatible. I don't know if this is an issue regarding the Phar version of PHP-CS-Fixer though.

> Should go in require dev then?

It won't work in the case PHP-CS-Fixer is used as a Composer dependency in a project that does not require `doctrine/annotations` itself, so I would either have it in `require`, or move it to `suggest` as @Slamdunk proposed, making it explicit that this must be manually installed if wanted.